clients/web/src/views/dashboard/console.tsx

```diff
@@ -23,12 +19,48 @@ export default function Console() {
     levels: [0, 1, 2, 3, 4],
   });
   const [filter, setFilter] = createStore<LogFilterObject>(initialFilters());
-  const filteredLogs = () => filterLogs(monitorData, filter, monitorData.logs);
+
+  const [filteredLogs, setFilteredLogs] = createSignal<LogEvent[]>([]);
```

Comment:
not sure I follow all the `untrack` happening in this code. Could you explain?
I'm thinking maybe this `createSignal` should be a store so we don't need to bail out so much...

Reply:
As explained in person this is meant to only trigger the effects based on changes to the corresponding reactive property.